src/modules/AddQR/info.json

```diff
@@ -12,6 +12,16 @@
         "type": "string",
         "desc": "input string to generate QR code",
         "default": "https://github.com/publiclab/image-sequencer"
+      },
+      "startingX": {
+        "type": "integer",
```

Comment:
Why not make these strings and use the util function `parseCornerCoordinateInput`? That will allow the user to input values like `50%` as well as `50`

Reply:
done